core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html

```diff
@@ -24,7 +24,10 @@ <h2 class="flex align-items-center gap-1 uppercase">
                 <div class="flex flex-row align-items-center align-content-between gap-2">
                     <div class="title flex flex-grow-1 flex-row align-items-center gap-2">
                         <ng-container
-                            *ngIf="variant.name === defaultVariantName; else editVariantNameTpl"
+                            *ngIf="
+                                variant.name === defaultVariantName || !vm.isExperimentADraft;
```

Comment:
This looks like a very specific condition that you want to put in the store.

For example `isExperimentADraft` means that there are other state, that should be reflected in the state.

Reply:
the other condition in particular refers to condition of each of the variants that can have the experiment that can be 1 to 3, is dynamic.